src/main/scala/com/mongodb/spark/config/MongoInputConfig.scala

```diff
@@ -38,8 +38,8 @@ package com.mongodb.spark.config
  *  - [[readPreferenceTagSetsProperty readPreference.tagSets]], the `ReadPreference` TagSets to use.
  *  - [[readConcernLevelProperty readConcern.level]], the `ReadConcern` level to use.
  *  - [[sampleSizeProperty sampleSize]], the sample size to use when inferring the schema.
- *  - [[splitKeyProperty splitKey]], the partition key to split the data.
- *  - [[maxChunkSizeProperty maxChunkSize]], the maximum chunk size when partitioning data from an unsharded collection.
+ *  - [[partitionerProperty partitioner]], the name of the partitioner to use to partition the data.
```

Comment:
I think we'll need a whole page on partitioners.  Will update configuring good catch.:)